beta/.husky/pre-commit

```diff
@@ -2,7 +2,6 @@
 . "$(dirname "$0")/_/husky.sh"
 
 cd beta
-# yarn generate-ids
-# git add -u src/pages/**/*.md
+yarn lint-heading-ids
```

Comment:
Why not fix them instead? Other scripts in this file fix things.

Reply:
- https://github.com/reactjs/reactjs.org/issues/4135#issuecomment-983729301

We did not want to fix the heading ids since other files without heading are modified while doing translations. Rather we wanted to just throw error so that we dont miss it in the main repo?